Sources/Workspace/Workspace.swift

```diff
@@ -932,20 +923,14 @@ public class Workspace {
                     return nil
                 }
 
+                let packagePath = path(for: managedDependency)
+
                 // The version, if known.
                 let version: Version?
-                let packagePath: AbsolutePath
-
-                // Construct the package path for the dependency.
                 switch managedDependency.state {
                 case .checkout(let checkoutState):
-                    packagePath = checkoutsPath.appending(managedDependency.subpath)
                     version = checkoutState.version
-                case .edited:
-                    packagePath = editablesPath.appending(managedDependency.subpath)
-                    version = nil
-                case .unmanaged(let path):
-                    packagePath = path
+                default:
```

Comment:
avoid using default because it hides future cases. Listing all cases is more code but it avoids subtle bugs.